pkg/plugin/run.go

```diff
@@ -250,26 +243,34 @@ func (e *AutoscaleEnforcer) handleAgentRequest(
 		}
 	}
 
+	status = 200
 	resp := api.PluginResponse{
 		Permit:  permit,
 		Migrate: migrateDecision,
 	}
-	return &resp, 200, nil
+
+	logger.Info(
+		"Handled agent request",
+		zap.Object("verdict", verdict),
+		zap.Int("status", 200),
```

Comment:
```suggestion
		zap.Int("status", status),
```

just aesthetics :)

Reply:
Oh, good catch - meant to do that. Thanks